py4DSTEM/preprocess/electroncount.py

```diff
@@ -74,15 +74,13 @@ def electron_count(datacube, darkreference, Nsamples=40,
         # Loop through frames
         for Rx in range(R_Nx):
             for Ry in range(R_Ny):
-                print_progress_bar(Rx*R_Ny+Ry+1, R_Ny*R_Nx, prefix=' Counting:', suffix='Complete',
-                                                                                 length = 50)
                 frame = datacube[Rx,Ry,:,:].astype(np.int16)    # Get frame from file
                 workingarray = frame-darkreference              # Subtract dark ref from frame
                 events = workingarray>thresh_bkgrnd             # Threshold electron events
                 events *= thresh_xray>workingarray
 
                 ## Keep events which are greater than all NN pixels ##
-                events = get_maximal_points(workingarray*events)
+                events = get_maxima_2D(workingarray*events)

```

Comment:
is this the same functionality, if so we should replace its use elsewhere

Reply:
The function name just changed